docs/atomic-trust.1.md

```diff
@@ -98,6 +103,10 @@ Remove a trust scope
 
     atomic trust delete docker.io
 
+Remove a trust scope but retain the sigstore configuration
+
+    atomic trust delete docker.io --sigstore
```

Comment:
I think you would need to say

 atomic trust delete docker.io --sigstore=false